doc/adoc/user_guide.adoc

```diff
@@ -154,8 +154,8 @@ manually. Once done, the upgrade process uses `zypper dup` and expects
 all required repositories to be setup correctly.
 
 Specify Migration Product::
-With the SLES15-Migration package a predefined product setup is
-already applied. That setting can be overwritten with one of the
+With the SLES15-Migration package an auto detected product is
+already applied. If that detection failed, the setting can be overwritten with one of the
 following product specifications:
 
 * SLES/15/x86_64
```

Comment:
Do we want to make the list of possible product names more complete ? I was thinking on also adding ```SLES_SAP/15/x86_64``` Furthermore I think it would be good to have a link to a page that shows the products suse provides. This is because in theory this upgrade process works for any of them. Thus the list we mention here is most probably incomplete. I'm not sure if suse provides such an information though. Thoughts ?

Reply:
I'm more than happy to put the link with all products suse provides.

Reply:
The PRD specifies supported upgrade path, but I am not certain this ends up in the documentation of not. But a search on suse.com/documentation in the admin guide for SLES 15 should reveal the information.

I do know that the product definitions as they are expected for this process have not been documented in the past.

Reply:
I couldn't come up with a one line location that summarizes this. So I suggest to keep the current state. Thanks @jesusbv for the list update